evap/staff/tools.py

```diff
@@ -368,3 +369,12 @@ def remove_user_from_represented_and_ccing_users(user, ignored_users=None, test_
             cc_user.cc_users.remove(user)
             remove_messages.append(_("Removed {} from the CC users of {}.").format(user.full_name, cc_user.full_name))
     return remove_messages
+
+
+def user_edit_link(user_id):
+    return format_html(
+        '<a href="{}" target=_blank><span class="fas fa-user-pen"> {}</span></a>'.format(
+            reverse("staff:user_edit", kwargs={"user_id": user_id}),
+            _("edit user"),
+        )
+    )
```

Comment:
This `.format`-inside-`format_html` construction will "abuse" `format_html` as `mark_safe`. Both format arguments should be properly escaped.

```suggestion
    return format_html(
        '<a href="{}" target=_blank><span class="fas fa-user-pen"> {}</span></a>',
        reverse("staff:user_edit", kwargs={"user_id": user_id}),
        _("edit user"),
    )
```